config/.eslintrc.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  env: {
+    es6: true,
+    es2020: true
+  },
+  extends: ['airbnb-base', 'prettier'],
+  plugins: ['@babel', 'prettier', 'prefer-import'],
+  parser: '@typescript-eslint/parser',
+  rules: {
+    // OFF
+    'import/prefer-default-export': 0,
+    'node/no-unsupported-features/es-syntax': 0,
+    'node/no-unsupported-features/es-builtins': 0,
+    camelcase: 0,
+    'class-methods-use-this': 0,
+    'linebreak-style': 0,
+    'new-cap': 0,
+    'no-param-reassign': 0,
+    'no-underscore-dangle': 0,
+    'no-use-before-define': 0,
+    'prefer-destructuring': 0,
+    'lines-between-class-members': 0,
+
+    // WARN
+    'prefer-import/prefer-import-over-require': [1],
+    'no-console': ['warn', { allow: ['assert'] }],
+    'no-debugger': 1,
+    'no-unused-vars': 1,
+    'arrow-body-style': 1,
+    'valid-jsdoc': [
+      1,
+      {
+        requireReturnDescription: false
+      }
+    ],
+    'prefer-const': 1,
+    'object-shorthand': 1,
+    'require-await': 1,
+
+    // ERROR
+    'no-unused-expressions': [2, { allowTaggedTemplates: true }]
+  }
+};
```

Comment:
IIRC there's something weird about `.eslintrc.js` existing in the root directory and being picked up by subprograms, bundlers, etc. so I'd prefer if we kept it in there instead of moving it like this. I would say all of `config/*` needs to get out of that folder (it was a mistake for me to try to consolidate in the first place) but that's a separate issue entirely.